### PR TITLE
feat(@vtmn/css): add price component

### PR DESCRIPTION
**Pull request by GaspardMathon**
## Pull Request checklist

🚨 Please review the [guidelines for contributing](../blob/main/CONTRIBUTING.md) to this repository.

- [X] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Don't request your main!
- [X] Check the commit's or even all commits' message styles matches our requested structure.
- [X] Check your code additions will fail neither code linting checks.
- [X] Check PR name, it must follow the https://www.conventionalcommits.org pattern
- [x] If it's a new component in CSS, ask for design review

## Description

- Add price component
- Four variants
- Three sizes

## Does this introduce a breaking change?

- No

❤️ Thank you!

### CODE REVIEW

File: packages/sources/css/src/components/price/src/index.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  color: var(--vtmn-semantic-color_content-primary-reversed);
+}
+
+.vtmn-price_variant--strikethrough {
+  background-color: transparent;
+  color: var(--vtmn-semantic-color_content-tertiary);
+  text-decoration: line-through;
+}
+
+/* Sizes */
```
Comment (lauthieb) [inline packages/sources/css/src/components/price/src/index.css:46]:
```suggestion
/* SIZES */
```

File: packages/sources/css/src/components/price/src/index.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  height: rem(28px);
+  font-size: var(--vtmn-typo_text-2-font-size);
+}
+
+/* SMALL */
+.vtmn-price_size--large {
+  min-height: rem(32px);
+  height: rem(32px);
+  font-size: var(--vtmn-typo_text-1-font-size);
+}
```
Comment (lauthieb) [inline packages/sources/css/src/components/price/src/index.css:67]:
```suggestion
/* Small */
.vtmn-price_size--small {
  min-height: rem(24px);
  height: rem(24px);
  font-size: var(--vtmn-typo_text-3-font-size);
}

/* Medium */
.vtmn-price_size--medium {
  min-height: rem(28px);
  height: rem(28px);
  font-size: var(--vtmn-typo_text-2-font-size);
}

/* Large */
.vtmn-price_size--large {
  min-height: rem(32px);
  height: rem(32px);
  font-size: var(--vtmn-typo_text-1-font-size);
}
```

File: packages/sources/css/src/components/price/src/index.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  font-family: var(--vtmn-typo_font-family);
+  font-size: var(--vtmn-typo_text-2-font-size);
+  font-weight: var(--vtmn-typo_font-weight--bold);
+  color: var(--vtmn-semantic-color_content-primary);
+  background-color: transparent;
+  text-align: center;
+  border-radius: rem(4px);
+}
+
+/* Variants */
```
Comment (lauthieb) [inline packages/sources/css/src/components/price/src/index.css:24]:
```suggestion
/* VARIANTS */
```

**Review by lauthieb — CHANGES_REQUESTED** [review]

**Review by GaspardMathon — COMMENTED** [review]
to edit

**Review by GaspardMathon — COMMENTED** [review]
modif

**Review by lauthieb — APPROVED** [review]

**Pull request merged**